modules/reference/pages/audit/audit-openscap-overview.adoc

```diff
@@ -118,7 +118,7 @@ mgr_check -vv
 ----
 +
 NOTE: If the {productname} daemon ([command]``rhnsd``) or [command]``osad`` are running on the client system, the action will be picked up by these services.
-To check if they are running, use:
+To check if they are running, on systemd-based systems use:
```

Comment:
```suggestion
On systemd-based systems, you can check if they are running with this command:
```

Steps in a procedure should always start with a verb (check if they are running), or a location (on systemd-based systems).

Reply:
ditto